week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,13 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = numberOfMentors + numberOfStudents;
+var studentPercentage = numberOfStudents / total;
+var studentRoughPercentage = Math.round(100 * studentPercentage);
+var mentorPercentage = numberOfMentors / total;
+var mentorRoughPercentage = Math.round(100 * mentorPercentage);
```

Comment:
Can you think of a way to use a function to make this more DRY (don't repeat yourself)?

Reply:
yes I will do.

Reply:
DRY implemented by using a function for displaying the message on the screen and a nested function for rounding the percentage.